teams/teams-ps/teams/Grant-CsTeamsMeetingTemplatePermissionPolicy.md

```diff
@@ -0,0 +1,141 @@
+---
+Module Name: MicrosoftTeams
+title: Grant-CsTeamsMeetingTemplatePermissionPolicy
+author: boboPD
+ms.author: pradas
+online version: https://learn.microsoft.com/powershell/module/teams/Grant-CsTeamsMeetingTemplatePermissionPolicy
+schema: 2.0.0
+---
+
+# Grant-CsTeamsMeetingTemplatePermissionPolicy
+
+## SYNOPSIS
+
+This cmdlet applies an instance of the TeamsMeetingTemplatePermissionPolicy to users or groups in a tenant.
```

Comment:
Empty lines between **SYNOPSIS** and the corresponding text will break the build process.

Reply:
Actually, it doesn't break the build process, but the blank line cause the Synopsis output from `Get-Help` to be blank. Please remove the blank line.